Startup/FullTime/FullTimeCheck.js

```diff
@@ -1,121 +1,151 @@
+					setVar("Preason_BadFullTime",true);
+                }else {
+					 sendVirtualAssistantMessage( random("You have been attending to your Mistress as required","You have been a devoted sub") +" "+ random("%GNMSlut%","%SlaveName%","Slave","Slave","Slave"));
+					 sendVirtualAssistantMessage(" I have a little treat for you! ");
+					 sendVirtualAssistantMessage(" Transferring gold.. ");
+					addGold(randomInteger(30,100));
+				}
+
+                setVar(VARIABLE_WEEKLY_SLAVE_VISITS, 0);
+                sendVirtualAssistantMessage("Let's see if you've been doing your chores like a good slave!");
+
+                if (getVar(VARIABLE_WEEKLY_CHORES_TIME) < getVar(VARIABLE_MIN_WEEKLY_CHORE_TIME)) {
+                    sendVirtualAssistantMessage("Bad boy!", "Bad girl!", "Bad slut!", "Bad dog!", "Bad slave!", "Bad sissy!");
+                    sendVirtualAssistantMessage("Bad behaviour is punished!");
+                    sendVirtualAssistantMessage("I just assigned you punishment points!");
+                    sendVirtualAssistantMessage("Do your chores!");
+                    addPunishmentPoints(200);
+					setVar("Preason_BadChores",true);
+                } else {
+                    sendVirtualAssistantMessage(random("Good boy!", "You have!", "Good little slut", "Good girl", "Good sissy", "Good slave"));
+                    sendVirtualAssistantMessage("Good behaviour is rewarded!");
+                    sendVirtualAssistantMessage("Transferring gold...");
+                    addGold(randomInteger(50, 200));
+                }
+
+                setVar(VARIABLE_WEEKLY_CHORES_TIME, 0);
+                setVar(VARIABLE_WEEKLY_CHORES_DONE, 0);
+
+                //TODO: Study and athlete mode
+            }
+        } 
+			
+		
+
+        if (isVar(VARIABLE_FULL_TIME_TRIAL_UNTIL) && getDate(VARIABLE_FULL_TIME_TRIAL_UNTIL).hasPassed()) {
+            delVar(VARIABLE_FULL_TIME_TRIAL_UNTIL);
+            sendVirtualAssistantMessage("Slave");
+            sendVirtualAssistantMessage("Your trial period has expired");
+            sendVirtualAssistantMessage("Meaning you now have a choice");
+            sendVirtualAssistantMessage("Stay full time or go back to part time");
+            sendVirtualAssistantMessage("Which is it?", false);
+            const answer = createInput();
+
+            while (true) {
+                if (answer.containsIgnoreCase("part")) {
+                    sendVirtualAssistantMessage("Back to part time then...");
+                    setPartTime();
+                    break;
+                } else if (answer.containsIgnoreCase("full")) {
+                    sendVirtualAssistantMessage("Excellent choice!");
+                    break;
+                } else {
+                    sendVirtualAssistantMessage("Part time or full time?");
+                    answer.loop();
+                }
+            }
+        }
+    } else if (!isVar(VARIABLE_NEXT_WEEK_CHECK)&&(getVar(VARIABLE_SLAVE_TYPE)==1)) {

```

Comment:
Use isFullTime()
Furthermore what exactly is this?

Reply:
this is the custom init session triggered by the code in the previous comment.  (its been a long time since I've seen this, I'm just trying to merge the function back in, not sure if I added it or it was orphaned.)